README.md

```diff
@@ -53,6 +53,45 @@ $ docker run -it --rm --name my-running-script -v "$PWD":/usr/src/app -w
 /usr/src/app node:4 node your-daemon-or-script.js
 ```
 
+## Verbosity
+
+By default Node.js Docker Image has NPM log verbosity set to `verbose` instead
+of the default `warn`. This is because of the way Docker is isolated from the
+host operating system and you are not guaranteed to be able to retrieve the
+`npm-debug.log` file when something breaks.
```

Comment:
Should we be a bit more explicit? Something like: "When `npm install` fails, it writes it's verbose log file to a log file on the filesystem. If npm fails during an install when building an image with `docker build`, this log file will be lost when the Docker image fails to build and the layer is thrown away. The Docker Working Group choose to be overly verbose during a build to provide an audit trail when an install fails."

(this is me thinking out loud :smile: )